tests/pass/concurrency/tls_lib_drop.rs

```diff
@@ -71,116 +73,7 @@ fn check_blocking() {
     thread::yield_now();
 }
 
-// This test tests that TLS destructors have run before the thread joins. The
-// test has no false positives (meaning: if the test fails, there's actually
-// an ordering problem). It may have false negatives, where the test passes but
-// join is not guaranteed to be after the TLS destructors. However, false
-// negatives should be exceedingly rare due to judicious use of
-// thread::yield_now and running the test several times.
-fn join_orders_after_tls_destructors() {
```

Comment:
Why did you just delete a test here?

Reply:
I don't know where this came from, must have been a mistake.